scripts/scratch/steps/0160-deploy-max-effective-balance-increaser.ts

```diff
@@ -0,0 +1,24 @@
+import { ethers } from "hardhat";
+
+import { deployWithoutProxy } from "lib/deploy";
```

Comment:
Also need to deploy upon upgrade (scripts/upgrade/steps/0100-deploy-contracts.ts)